src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs

```diff
@@ -100,10 +100,6 @@ public interface IAppConfig
         /// </summary>
         IDictionary<string, string> ExtraQueryParameters { get; }
```

Comment:
What about ExtraQueryParameters  ?

Reply:
@bgavrilMS what about them?

Reply:
I was just wondering if they make sense at this level. I think they do...

Reply:
@bgavrilMS Oh...I see. you mean should we leave them here or move them like claims? I agree with you, I think they make sense to leave here on the application object. @jmprieur thoughts?

Reply:
They do make sense both at the app level and the acquire token level
for instance at the app level: slices
at the acquiretoken level: mkt=fr-fr